src/jsonld/jobPosting.tsx

```diff
@@ -53,13 +54,20 @@ const buildBaseSalary = (baseSalary: MonetaryAmount) => `
     ${baseSalary.currency ? `"currency": "${baseSalary.currency}",` : ''}
     "value": {
       "@type": "QuantitativeValue",
-      ${baseSalary.value ? `"value": "${baseSalary.value}",` : ''}
+      ${
```

Comment:
I spotted two small bugs here if you could change this block to be:

```
const buildBaseSalary = (baseSalary: MonetaryAmount) => `
  "baseSalary": {
    "@type": "MonetaryAmount",
    ${baseSalary.currency ? `"currency": "${baseSalary.currency}",` : ''}
    "value": {
      ${
        baseSalary.value
          ? Array.isArray(baseSalary.value)
            ? `"minValue": "${baseSalary.value[0]}", "maxValue": "${baseSalary.value[1]}",`
            : `"value": "${baseSalary.value}",`
          : ''
      }
      ${baseSalary.unitText ? `"unitText": "${baseSalary.unitText},"` : ''}
      "@type": "QuantitativeValue"
    }
  },
`;
```
Changes made were:

- Removed `\n\t` invalid JSON characters
- Moved `@type` to be last so that the trailing `,` are more predictable from `value` and `minValue` / `maxValue`

Reply:
Thanks for reviewing! This is now fixed in [`ff2bb63` (#601)](https://github.com/garmeeh/next-seo/pull/601/commits/ff2bb636a715aa35bd03f1af365751c9e2f8c569).